api/draft_registrations/views.py

```diff
@@ -68,12 +69,15 @@ def get_queryset(self):
         if user.is_anonymous:
             raise exceptions.NotAuthenticated()
         # Returns DraftRegistrations for which a user is a contributor
-        return user.draft_registrations_active
-
+        return DraftRegistration.objects.filter(
+            Q(_contributors=user) &
+            Q(deleted__isnull=True) &
+            (Q(registered_node__isnull=True) | Q(registered_node__deleted__isnull=False)),
+        )
```

Comment:
Why the explicit query instead of reusing the `user.draft_registrations_active`?